src/raftstore/store/fsm/apply.rs

```diff
@@ -1162,6 +1163,7 @@ impl ApplyDelegate {
         let key = keys::data_key(key);
         self.metrics.size_diff_hint += key.len() as i64;
         self.metrics.size_diff_hint += value.len() as i64;
+        self.metrics.keys_diff_hint += 1;
```

Comment:
this is not right, we regards default record and write record as one key.

Reply:
@Connor1996 then maybe we can only calculate defalut_cf here?

Reply:
maybe calculate both `defalut_cf` and `write_cf`